prov/usnic/src/usdf_ep_dgram.c

```diff
@@ -508,6 +508,7 @@ int usdf_dgram_fill_dom_attr(uint32_t version, const struct fi_info *hints,
 
 	defaults = dgram_dflt_domain_attr;
 	ret = usdf_domain_getname(version, dap, &defaults.name);
+        printf("Got USDF DGRAM domain name: %s (ret=%d)\n", defaults.name, ret);
```

Comment:
indent does not appear to match?

Reply:
Oops -- spaces vs. tabs there.  Also, that's a debugging printf that slipped through.  Removed.